vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj

```diff
@@ -77,7 +77,8 @@
         RN_EXPORT=;
         %(PreprocessorDefinitions)
       </PreprocessorDefinitions>
-      <AdditionalOptions>%(AdditionalOptions) /Zc:strictStrings</AdditionalOptions>
+      <ControlFlowGuard>Guard</ControlFlowGuard>
```

Comment:
I had thought that @JunielKatarn had enabled this for release mode.

https://github.com/microsoft/react-native-windows/blob/933eee37877a1c50abb523b8ce90c44e77ee496b/vnext/PropertySheets/Release.props

Reply:
I'm not seeing `Release.props` imported anywhere actually 🤔. I wonder if some build logic change may have caused this to no longer be included.